### PR TITLE
feat: #76 Add output to main docker workflow

### PR DESCRIPTION
**Pull request by AlexAxthelm**
Closes: #76

### CODE REVIEW

**Comment by AlexAxthelm** [comment]
See it in action: https://github.com/RMI-PACTA/workflow.scenario.preparation/actions/runs/8722260802

File: .github/workflows/docker.yml [hunk 1]
```diff
@@ -38,6 +38,10 @@ on:
         required: false
         default: true
         type: boolean
+    outputs:
+      full-image-name:
+        description: "Full pushed image name including host/registry, name, and tag"
+        value: ${{ jobs.build-docker-image.outputs.full-image-name }}
```
Comment (cjyetman) [inline .github/workflows/docker.yml:44]: does `build-docker-image.outputs.full-image-name` have to be specified in the `build-docker-image` job below too?

Comment (cjyetman) [reply .github/workflows/docker.yml:44]: don't know, just asking

**Review by cjyetman — APPROVED** [review]

File: .github/workflows/docker.yml (hunk 1 above)
Comment (AlexAxthelm) [reply .github/workflows/docker.yml:44]: It is specified as an output already in that workflow. I just never propagated it up to the main calling workflow.

**Pull request merged**